content/02.body.md

```diff
@@ -2,13 +2,65 @@
+Consequently, degree-based predictions are more likely nonspecific, meaning the same set of predictions would appear to perform well for different tasks.
+However, depending on the prediction task and the desired types of predictions, edge predictions between very high degree nodes may be uninsightful or nonspecific.
+Model evaluation can also be complicated, as nonspecific or trivial predictions need not be incorrect, even if they are not the desired outputs of the predictive model.
+Rather, degree-based features must be included in the interpretation of predictions in order to disentangle desired from non-desired effects and to more effectively evaluate and compare predictive models.
+
+![**A.** Degree distributions of networks with and without degree bias can be very different.
+  Protein-protein interaction (PPI) literature-derived network from STRING (Szklarczyk et al.) [@doi:10.1093/nar/gky1131], high-throughput-derived networks from Rual et al. [@doi:10.1038/nature04209] and Rolland et al. [@doi:10.1016/j.cell.2014.10.050].
+  Transcription factor-target gene (TF-TG) literature-derived network from Han et al. [@doi:10.1093/nar/gkx1013] and high-throughput network from Lachmann et al. [@doi:10.1093/bioinformatics/btq466].
+  BioRxiv data from Rxivist [@doi:10.1101/515643; @doi:10.5281/zenodo.2566421] had edges split between those existing before and during/after 2018.
+  **B.** Uniform random sampling edges produces node degrees that are linearly correlated with those in the unsampled network.
+  Shown are the same PPI and TF-TG networks as in subfigure A.
+  70% of literature edges were sampled with uniform probability.
+  **C.** Node degree in a literature-derived network is not linearly correlated with degree in a systematically-derived network.
+  Assuming that the inclusion of an edge in the systematic network is independent of node degree, the systematic degree should be linearly correlated with the true number of relationships.
+  Edges found in literature-derived networks represent a non-random sample from systematically-derived edges.
+  Shown are the same PPI and TF-TG networks as in subfigure A.
+  **D.** Degree distribution can vary greatly between networks, even between the source and target degree distributions within the same network.
+  Shown are six networks from Hetionet [@doi:10.7554/eLife.26726].
+  **E.** Common edge-prediction features are correlated with node degree.
+  Shown are five features (Supplemental table {@tbl:edge-prediction}) computed on STRING protein-protein interaction network [@doi:10.1093/nar/gky1131] and the product of the node pair's source and target degrees.
+  All five features show a positive relationship with degree, though the magnitude of this correlation is highly variable.
+](https://github.com/zietzm/xswap-analysis/raw/new-plots/img/fig1.bias_features_combined.png){#fig:degree width="100%"}
```

Comment:
Best if link to a commit hash rather than branch, since the content pointed to by a branch can CHANGE